docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+|PPA&#8209;50|TBD: Placeholder for single-precision floating performance per MHz.
+|PPA&#8209;60|TBD: Placeholder for double-precision floating performance per MHz.
+|===
+
+== Interface requirements
+
+=== Memory bus
+
+|===
+
+|MEM&#8209;10|CVA6 memory interface shall comply with AXI5 specification including the Atomic_Transactions property support as defined in [AXI] section E1.1.
+|MEM&#8209;20|CVA6 AXI memory interface shall feature user bit extensions on the data bus (`WUSER` and `RUSER` as per [AXI]) in connection with the L1I and L1WTD datasize extensions, with a number of user bits greater or equal to 0.
+|===
+
+The interface usually complies with AXI4. However, Atomic_Transactions is only defined in AXI5. For the sake of clarity, we do not use the AXI5-Lite interface.
+
+=== Debug
+
+|===
+
+|DBG&#8209;10|CVA6 shall implement both the Abstracted Command and Execution based features outlined in chapter 4 of [RVdbg].
+
+|===
+In addition, there can be an external debug module, not in the scope of the IP.
+
+=== Interrupts
+
+|===
+
+|IRQ&#8209;10|CVA6 shall implement interrupt handling registers as per the RISC-V privilege specification and interface with a CLINT implementation.
+
+|===
+=== Coprocessor interface
+
+|===
+
+|XIF&#8209;10|To extend the supported instructions, CVA6 shall have a coprocessor interface that supports the “Issue”, “Commit” and “Result” interfaces of the [CV-X-IF] specification.
+
+|===
+
+The goal is to have a compatible interface between CORE-V cores (CVA6, CV32E40X…). The feasibility still needs to be confirmed; including the speculative execution.
+
+CVA6 can interface with several coprocessors simultaneously through a specific external feature implemented on the CV-X-IF interface.
+
+=== Multi-core interface
+
+|===
+
+|TRI&#8209;10|CVA6 shall have the Transaction-Response Interface (TRI) needed to interface with the P-Mesh coherence system of OpenPiton, according to [OpenPiton].
+
+|===
+
+=== Design rules
+
+As different teams have different design rules and to ease the integration in FPGA and ASIC design flows:
+
+|===
+
+|RUL&#8209;10|CVA6 should have a configurable reset signal: synchronous/asynchronous, active on high or low levels.
+|RUL&#8209;20|For certain FPGA targets, CVA6 should as an *option* remove the reset in the RTL code.
+|RUL&#8209;30|CVA6 shall be a super-synchronous design with a single clock input.
+|RUL&#8209;40|CVA6 should not include multi-cycle paths.
+|RUL&#8209;50|CVA6 should not include technology-dependent blocks.
+|===
+[FPGAreset] provides background information about the RUL-20 requirement.
+
+If technology-dependent blocks are used, e.g. to improve PPA on certain targets, the equivalent technology-independent block should be available. Parameters can be used to select between the implementations.
```

Comment:
Wasn't clear what "technology-independent block should be available" is requiring.

Reply:
Duplicate remark.